packages/deployment/app/kes.override.js

```diff
@@ -379,18 +390,37 @@ class UpdatedKes extends Kes {
         reject(err);
       });
     });
-  };
+  }
+
+  /**
+   * unzip a given zip file to the given destination
+   *
+   * @param {string} filename - the zip file to extract
+   * @param {string} dst - the destination to extract the file
+   * @returns {Promise} the path of the extracted zip
+   */
+  extractZipFile(filename, dst) {
+    // create the destination folder it doesn't exist
+    fs.mkdirpSync(dst);
```

Comment:
Since this already returns a promise, `fs.mkdirp(dst)` could / should be used instead of the sync version.